testing/fuchsia/test_suites.yaml

```diff
@@ -1,27 +1,25 @@
 # This configuration file specifies several test suites with their package and
 # test command for femu_test.py.
 
-- package: flutter_runner_tzdata_tests-0.far
-  test_command: run-test-component fuchsia-pkg://fuchsia.com/flutter_runner_tzdata_tests#meta/flutter_runner_tzdata_tests.cmx
+- package: testing_tests-0.far
```

Comment:
I re-ordered the tests in this file deliberately.

This way they run in order of complexity, with simpler/fewer-dependencies tests running first. the last test is the most complex conceptually and has the most dependencies.

This helps a lot when debugging test failures.

The net-effect of the reordering is to just delete "flutter_runner_scenic_tests".

Reply:
Nevermind, reverted this